indigo_app/templates/indigo_api/work_overview.html

```diff
@@ -185,6 +185,15 @@ <h4 class="text-muted">{{ work.frbr_uri }}</h4>
                         {{ work.consolidation_note }}
                       </dd>
                     {% endif %}
+
+                    {% if work.disclaimer %}
+                      <dt>
+                        {% trans 'Disclaimer' %}
+                      </dt>
+                      <dd>
+                        {{ work.disclaimer|linebreaks }}
```

Comment:
How will the renderer handle linebreaks?

Reply:
do you mean PDF?

Reply:
HTML
![image](https://user-images.githubusercontent.com/32566441/213426016-73e424a4-2a34-4bc9-b9a1-748fb3be6f71.png)


PDF
![image](https://user-images.githubusercontent.com/32566441/213425797-2909a3b6-31fe-453c-b363-5fbe14482412.png)

Reply:
I might totally be misunderstanding the question :sweat_smile:

Reply:
if we handle linebreaks in one place, it must be handled the same way everywhere (html, pdf, overview page, etc.). I'm just checking that we do that.

Reply:
The only difference seems to be that the PDF doesn't render blank linebreaks. Should we add support for that now?